start.py

```diff
@@ -242,32 +248,22 @@ def cm_server_not_dead(primary_node):
     logger.info('Updating CM server configurations ...')
     deployment.update_cm_config(configs={'manages_parcels': True})
 
-    if args.sdc_version:
+    if args.sdc_build:
+        # We install StreamSets DataCollector using local repo /opt/cloudera/parcel-repo.
+        # Set file and folder permissions correctly.
+        commands = ['chown cloudera-scm:cloudera-scm /opt/cloudera/csd',
+                    'chown cloudera-scm:cloudera-scm /opt/cloudera/parcel-repo',
+                    'chown cloudera-scm:cloudera-scm /opt/cloudera/csd/STREAMSETS*.jar',
+                    'chmod 644 /opt/cloudera/csd/STREAMSETS*.jar',
+                    'chown cloudera-scm:cloudera-scm /opt/cloudera/parcel-repo/STREAMSETS_*']
+        primary_node.execute('; '.join(commands))
```

Comment:
Might wanna join with `&&` to make sure failing commands kill it.

Reply:
Done.